src/styles/antisnake.scss

```
@@ -0,0 +1,47 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  justify-content: center;
+
+  &__block {
+    height: 300px;
+    min-width: 300px;
+    display: grid;
+    justify-content: center;
+    align-items: center;
+
+    &--1 {
+      background-color: rgb(255, 0, 0);
+    }
+    
+    &--2 {
+      background-color: rgb(204, 0, 0);
+    }
+    
+    &--3 {
+      background-color: rgb(153, 0, 0);
+    }
+    
+    &--4 {
+      background-color: rgb(102, 0, 0);
+    }
+    
+    &--5 {
+      background-color: rgb(51, 0, 0);
+    }
+    
+    &--6 {
+      background-color: rgb(0, 0, 0);
+    }
```

все цвета можно записать в одну формулу (футкцию @for), что бы укоротить код и по возможности если нам понадобиться в будущем поменять цвет то мы просто его изменим лишь в одном месте. 
```suggestion
  @for $i from 1 through 6 {
    &:nth-child(#{$i}) {
      background-color: rgb(255 - 51 * ($i - 1), 0, 0);
    }
  }
```
у меня эта функция выглядит не лучшим образом так что подумай над своей.. но что-то похожее на мою.. можешь усовершенствовать))

---

виправила на формулу) дякую